rbmc-cfam-daemon/src/bmc_cfam.hpp

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+
+#include "cfam_fields.hpp"
+
+#include <algorithm>
+#include <map>
+#include <set>
+
+/**
+ * @class BMCCFAM
+ *
+ * The base class representing a CFAM-S on a BMC card, where the four
+ * scratchpad registers in the Gemini Mailbox block of the CFAM are
+ * used to store redundant BMC information.
+ *
+ * Children of this base class are local CFAM and sibling CFAM classes.
+ */
+class BMCCFAM
+{
+  public:
+    /**
+     * @brief The fields stored in the mailbox scratchpad registers
+     * in CFAM-S chips off of a BMC.
+     */
+    enum class Field
```

Comment:
I just want to confirm—BMCCFAM is little-endian, correct? And do we account for that when reading bit values from the register?

Reply:
CFAMs are big endian, and so is the driver.  What I have here definitely works with the HW.

Reply:
> CFAMs are big endian

Thanks for correcting me.